meshtastic/interfaces/ble/notifications.py

```
@@ -228,3 +265,741 @@ def _get_callback(self, characteristic: str) -> Callable[[Any, Any], None] | Non
+            for characteristic in characteristics:
+                try:
+                    client.stop_notify(
+                        characteristic,
+                        timeout=NOTIFICATION_START_TIMEOUT,
+                    )
+                except Exception:  # noqa: BLE001 - stale rollback must stay best effort
+                    logger.debug(
+                        "Error stopping stale notification subscription for %s",
+                        characteristic,
+                        exc_info=True,
+                    )
```

Ruff (0.15.6)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)